docs/Monitored_Only_ZFS.md

```diff
@@ -0,0 +1,208 @@
+    sudo -s
+    zpool create oss1 -o cachefile=none raidz2 /dev/disk/by-id/ata-VBOX_HARDDISK_VB6f41df02-0d5d2a15 /dev/disk/by-id/ata-VBOX_HARDDISK_VB06b563a9-2539af7b  /dev/disk/by-id/ata-VBOX_HARDDISK_VBb2ec79fb-b900b724 /dev/disk/by-id/ata-VBOX_HARDDISK_VBe2585d74-5267121b -f
+    mkfs.lustre --reformat --failover 10.73.20.22@tcp  --ost --backfstype=zfs --fsname=zfsmo --index=0 --mgsnode=10.73.20.11@tcp oss1/ost00
+    zfs compression=on oss1
+    mkdir -p /lustre/zfsmo/ost00
+    mount -t lustre oss1/ost00 /lustre/zfsmo/ost00
+
+    vagrant ssh oss2
+    sudo -s
+    zpool create oss2 -o cachefile=none raidz2 /dev/disk/by-id/ata-VBOX_HARDDISK_VBb7776d10-aba16176 /dev/disk/by-id/ata-VBOX_HARDDISK_VB6657241f-bab2ffed  /dev/disk/by-id/ata-VBOX_HARDDISK_VB0ebfc209-9bbf80c4 /dev/disk/by-id/ata-VBOX_HARDDISK_VB51449aa2-95df9cdc -f
+    mkfs.lustre --reformat --failover 10.73.20.21@tcp  --ost --backfstype=zfs --fsname=zfsmo --index=1 --mgsnode=10.73.20.11 oss2/ost01
+    zfs compression=on oss2
+    mkdir -p /lustre/zfsmo/ost01
+    mount -t lustre oss2/ost01 /lustre/zfsmo/ost01
+```
+
+After all the commands for each had run successfully, use the IML GUI to scan for the filesystem:
+    Configuration -> Servers -> Scan for Filesystem.  Use all servers
+
+If Succussfull, in the IML GUI, the filesystem will be available.
+
+## Setting up Clients
+In your vagrant folder, run the following script to prepare both client c1 and c2:
+```
+vagrant sh -c '\
+sudo yum-config-manager --add-repo https://copr.fedorainfracloud.org/coprs/managerforlustre/lustre-client/repo/epel-7/managerforlustre-lustre-client-epel-7.repo && \
+sudo yum-config-manager --add-repo https://downloads.hpdd.intel.com/public/e2fsprogs/latest/el7/ && \
+sudo sed -i -e "1d" -e "2s/^.*$/[e2fsprogs]/" -e "/baseurl/s/,/%2C/g" -e "/enabled/a gpgcheck=0" /etc/yum.repos.d/downloads.hpdd.intel.com_public_e2fsprogs_latest_el7_.repo && \
+sudo yum clean metadata && \
+sudo yum -y install lustre-client && \
+sudo reboot' c1 c2
+```
+Both clients are now running the lustre-client software and are ready to be mounted. To mount a client, do the following:
+1. ssh into the client
+```
+vagrant ssh c1
+```
+2. Create a mount directory (it's a good idea to use the same name as the filesystem you created).
+```
+sudo mkdir -p /mnt/zfsmo
+```
+3.  Note the NID for the MGS. You can get this by looking at the lustre network interface in the GUI on the server detail page for mds1 or you can run the following command:
+```
+vagrant ssh mds1
+lctl list_nids
+# 10.73.20.11@tcp # Notice this is the lustre network interface
+```
+4. Note the NID for the MDS. You can get this by looking at the lustre network interface in the GUI on the server detail page for mds2 or you can run the following command:
+```
+vagrant ssh mds2
+lctl list_nids
+10.73.20.12@tcp # Notice this is the lustre network interface
+```
+5. Mount the lustre filesystem:
+```
+sudo mount -t lustre 10.73.20.11@tcp:10.73.20.12@tcp:/fs /mnt/zfsmo
```

Comment:
Again, all of this business of finding NIDs is unnecessary and you should be able to use host names instead of IP addresses here.

Reply:
Or just use the mount output from the GUI.

Reply:
Fixed